src/bin/utils/parsers.ts

```diff
@@ -72,4 +101,7 @@ export {
   parseSeedNodes,
   parseCoreCount,
   parseSecretPath,
+  parseProviderIdList,
+  parseIdentityId,
+  parseProviderId,
```

Comment:
Get your exports in the same order as your definitions above.

Reply:
Fixed